### DISCUSSION TITLE
Should PrivateBin be more customizable and if so, what customization options would you need most?

### DISCUSSION THREAD

**Discussion by rugk** [PrivateBin/PrivateBin#1321] [Polls]
In https://github.com/PrivateBin/PrivateBin/issues/639 it was requested that PrivateBin should be more configurable/themable/brandable without having to edit and maintain templates?

However, we cannot make everything customizable in an easy and secure way, so we'd like to know what you need most?

The poll also includes items [which already are customizable](https://github.com/PrivateBin/PrivateBin/wiki/Configuration), but for reference, please also vote on these, if you think they should be/stay customizable.

**Comment by nain-F49FF806** [comment]
I would like an option to disable download button for media attachments / pastes. 
This would be partially possible with user.css. 

I think this would be a good configurable feature on its own too. For example, when burn on read is on,
- Downloading the paste as file should be disabled
- Downloading of attachments should be disabled

As the poster likely intends to reduce persistence for such pastes.  
Sure, third-party clients would not be affected. 
That's okay. It's only a convenience feature, gently nudging against persistence.

**Reply by rugk** [reply]
Maybe create a new feature request for that?

**Reply by chrish112** [reply]
The funny thing is I worked around this by flat out removing the buttons but that doesn't allow me the option to use anymore. There are specific situations where I want some people to use it for a customer but others not.

**Reply by rugk** [reply]
> There are specific situations where I want some people to use it for a customer but others not.

Then you need to add custom JS or HTML to show it in one case or in another. Such special template and design adjustments like this are in your responsibility to develop, then.

**Comment by chrish112** [comment]
My big one is customizing the website. I have it configured to more or less what I want but I don't know if it will break when I upgrade.

**Comment by voidquark** [comment]
Agree customizing icon/logo will be amazing.

**Comment by Nimothenicefish** [comment]
+1 for customizing icon/logo

**Comment by sebatminalea** [comment]
Same here! icon/logo customization would be cool!
